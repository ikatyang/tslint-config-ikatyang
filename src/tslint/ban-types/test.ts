// good

declare const good1: string;

// bad

declare const bad1: Boolean;
declare const bad2: Number;
declare const bad3: String;
declare const bad4: Object;
