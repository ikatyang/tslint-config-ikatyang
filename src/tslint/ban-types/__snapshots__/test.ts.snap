// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

declare const good1: string;

// bad

declare const bad1: Boolean;
                    ~~~~~~~  [Don't use 'Boolean' as a type. Use boolean instead.]
declare const bad2: Number;
                    ~~~~~~  [Don't use 'Number' as a type. Use number instead.]
declare const bad3: String;
                    ~~~~~~  [Don't use 'String' as a type. Use string instead.]
declare const bad4: Object;
                    ~~~~~~  [Don't use 'Object' as a type. Use object instead.]

`;
